etc/kayobe/kolla.yml

```diff
@@ -116,10 +116,19 @@ kolla_build_blocks:
   sahara_engine_footer: |
     # Bring in hack to the vanilla plugin
     RUN pip install -U --no-deps git+https://github.com/stackhpc/sahara.git@stackhpc/7.0.2.1
+  monasca_api_footer: |
+    # Rebase on top of rocky
```

Comment:
nit: Rebase might confuse people with git terminology. Please could we say what this is fixing and include a link to the upstream patch?